rllib/env/utils/infinite_lookback_buffer.py

```diff
@@ -34,6 +35,77 @@ def __init__(
         self.space_struct = None
         self.space = space
 
+    def __eq__(
```

Comment:
dumb question: why do we need this method?

Reply:
Just for testing purposes to make comparisons of instances before writing and after writing.